src/main/kotlin/com/cognifide/gradle/aem/pkg/tasks/PackageCompose.kt

```diff
@@ -391,7 +394,7 @@ open class PackageCompose : ZipTask() {
             }
 
             into("${Package.JCR_ROOT}/$effectivePath") { spec ->
-                spec.from(other)
+                spec.from(other.composedFile)
```

Comment:
for some reason when using `from(composeTask)` OAKPAL_DIR was also being a part of effective package; bug fix